test/web-platform-tests/to-upstream/geometry/dommatrix.html

```diff
@@ -16,7 +16,7 @@
     dma = dma instanceof DOMMatrix ? dma : DOMMatrix.fromMatrix(dma);
     dmb = dmb instanceof DOMMatrix ? dmb : DOMMatrix.fromMatrix(dmb);
 
-    assert_array_approx_equals(arrayFromDm(dma), arrayFromDm(dmb), 0.00000000001);
+    assert_array_approx_equals(arrayFromDm(dma), arrayFromDm(dmb), 0.000001);
```

Comment:
If anyone has suggestions on how to intuitively preform these calculations with more precision, I'm open to hearing how to do it. Since the browsers are preforming matrix multiplication with native libraries, the precision varies in some cases but is negligible.